rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2504,6 +2504,17 @@ public long getNextPegoutCreationBlockNumber() {
         return activations.isActive(RSKIP271) ? provider.getNextPegoutHeight().orElse(0L) : 0L;
     }
 
+    public int getQueuedPegoutsCount() {
+        if (activations.isActive(RSKIP271)) {
+            try {
+                return provider.getReleaseRequestQueueSize();
+            } catch (IOException e) {
+                logger.error("Unexpected error accessing storage while attempting to get the ReleaseRequestQueueSize", e);
+            }
```

Comment:
I'm not sure it's a good idea to catch the exception here, if there's an error reading from the storage the function will return `0` which could be misleading. I'd suggest removing the try/catch block and letting the Bridge `execute` method handle it

Reply:
Resolved